app/Models/Share.php

```diff
@@ -43,6 +43,10 @@ public static function load($filename) {
 			$share_options['type'] = $share_type;
 			self::register($share_options);
 		}
+
+		uasort(self::$list_sharing, function ($a, $b) {
+			return $a->name() < $b->name() ? -1 : 1;
```

Comment:
```suggestion
			return strcmp($a->name(), $b->name());
```

Would the above work? I think it would be cleaner, and takes into account equality (no impact in this case though)

Reply:
I'd rather use this:
```
		uasort(self::$list_sharing, function ($a, $b) {
			return $a->name() <=> $b->name();
		});
```

But it's not possible at the moment ;)

Reply:
@Alkarex I've changed it.